### PR TITLE
Convert to Gradle

### PR DESCRIPTION
**Pull request by scerruti**
Convert build of plugin to Gradle.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (s5bug) [inline .idea/misc.xml]: This file shouldn't be tracked

File: .idea/modules.xml [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <component name="ProjectModuleManager">
```
Comment (s5bug) [inline .idea/modules.xml]: Neither should this

File: .idea/vcs.xml [hunk 3]
```diff
@@ -2,5 +2,6 @@
 <project version="4">
```
Comment (s5bug) [inline .idea/vcs.xml]: Or this

File: discord-intellij.iml [hunk 4]
```diff
@@ -1,22 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (s5bug) [inline discord-intellij.iml]: The IML should not be tracked either

File: src/main/java/com/tsunderebug/discordintellij/ToggleAction.java [hunk 5]
```diff
@@ -4,7 +4,7 @@
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.util.IconLoader;
 
-import javax.swing.Icon;
+import javax.swing.*;
```
Comment (s5bug) [inline src/main/java/com/tsunderebug/discordintellij/ToggleAction.java]: Fix this wildcard

**Review by s5bug — CHANGES_REQUESTED** [review]

**Comment by scerruti** [comment]
@TsundereBug requested changes made.

**Review by s5bug — APPROVED** [review]

**Comment by s5bug** [comment]
Keeping open for Vatuu RPC change

**Pull request merged**